src/include/execution/compiler/operator/operator_translator.h

```diff
@@ -146,6 +154,42 @@ class OperatorTranslator : public ColumnValueProvider {
    */
   virtual void BeginPipelineWork(const Pipeline &pipeline, FunctionBuilder *function) const {}
 
+  /**
+   * Function to initialize relevant countfers.
```

Comment:
Typo: counters

Reply:
Same with RecordCounters below.
And optionally if you want to throw a period at the end of pipeline's parameter comment that'd be nice.